uncrustify.cfg

```diff
@@ -83,11 +83,12 @@ sp_after_ptr_star_func		= remove
 sp_after_byref_func			= remove
 sp_before_ptr_star_func		= force
 sp_before_byref_func		= force
+sp_after_comma              = force
```

Comment:
Hmmm, I should point out you are using spaces to indent, while the file is using tabs.

Reply:
I am using editor with tab = 4 spaces and converting newly entered tab to spaces. It complies with project codestyle. With this settings file looks ok.  Github uses other tab settings and shows lines unaligned. The best thing we can do here is to convert all tabs to 4 spaces but this must be done in another PR.

Reply:
I don't mind you convert it later, however it would make sense to use tabs in the current PR, isn't it?

Reply:
converted to tabs